pkgs/development/tools/prospector/default.nix

```diff
@@ -60,6 +54,17 @@ buildPythonApplication rec {
     pytestCheckHook
   ];
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace 'requirements-detector = "^0.7"' 'requirements-detector = "*"' \
+      --replace 'pep8-naming = ">=0.3.3,<=0.10.0"' 'pep8-naming = "*"' \
+      --replace 'mccabe = "^0.6.0"' 'mccabe = "*"'
+  '';
```

Comment:
I think it makes more sense to keep this close to `src`.

Reply:
I'm trying follow more or less `nixpkgs-hammering`'s style recommendation on the ordering.